### PR TITLE
Added badge notification component

### PR DESCRIPTION
**Pull request by Pallavi-PH**
Added badge notification component

Screenshot 2020-09-21 at 11 21 45 PM

### CODE REVIEW

File: scss/core/abstracts/_variables.scss [hunk 1]
```diff
@@ -746,6 +746,9 @@ $badge-padding-y:                   .4em !default;
 $badge-padding-x:                   1em !default;
 $badge-border-radius:               10rem !default;
 
+$badge-notification-border: 2px solid $red-100 !default;
+$badge-notification-color: $red-100 !default;
```
Comment (is-ashish) [inline scss/core/abstracts/_variables.scss:750]: @Pallavi-PH, Is badge-notification have only  `$red-100` color ?? As of now.

Comment (Pallavi-PH) [reply scss/core/abstracts/_variables.scss:750]: Yes, currently we have only one variation if badge notification.

Comment (is-ashish) [reply scss/core/abstracts/_variables.scss:750]: Okay thanks

**Review by is-ashish — APPROVED** [review]

**Review by avishnu — APPROVED** [review]
LGTM

**Pull request merged**